### PR TITLE
cleaned up integration-cli/docker_api_images_test.go

### PR DESCRIPTION
**Pull request by ZJaffee**
As part of "Proposal: use of checkers on Integration test #16756"

### CODE REVIEW

**Comment by vdemeester** [comment]
@ZJaffee As stated by the build you need to format the modified files, with `gofmt -s -w`

File: integration-cli/docker_api_images_test.go [hunk 1]
```diff
@@ -23,59 +24,45 @@ func (s *DockerSuite) TestApiImagesFilter(c *check.C) {
...
-	if images := getImages("utest"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
-	if images := getImages("utest*"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
-	if images := getImages("*5000*/*"); len(images[0].RepoTags) != 1 {
-		c.Fatal(errMsg)
-	}
+	c.Assert(len(getImages("utest*/*")[0].RepoTags),checker.Not(checker.Equals),2,Commentf(errMsg))
```
Comment (vdemeester) [inline integration-cli/docker_api_images_test.go]: Here and everywhere on the file : s/Commentf/check.Commentf/

File: integration-cli/docker_api_images_test.go [hunk 2]
```diff
@@ -23,59 +24,45 @@ func (s *DockerSuite) TestApiImagesFilter(c *check.C) {
...
+	c.Assert(len(getImages("utest")[0].RepoTags),checker.Not(checker.Equals),1,Commentf(errMsg))
```
Comment (vdemeester) [inline integration-cli/docker_api_images_test.go]:
```go
c.Assert(getImages("utest"), checker.Not(checker.HasLen), 1, check.Commentf(errMsg))
```

Same above.

**Comment by calavera** [comment]
I'm very sorry but I'm going to close this PR until the errors are resolved. Please, feel free to send it again after testing that every test still works and the code is valid. `make test` and `make validate` are your friends.

As a rule of thumb, before opening all your PRs again, it will be easier to get them merged if you focus in one at a time, that way reviewers don't need to point the same errors over several changes.

**Pull request closed without merging**